hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -799,11 +804,42 @@ public void incNumKeyDeleteFails() {
     numKeyDeleteFails.incr();
   }
 
+  public void incNumKeyDeleteFails(int count) {
+    numKeyDeleteFails.incr(count);
+  }
+
+  public void incNumKeyRetentionDeleteFails(int count) {
+    numKeyRetentionDeleteFails.incr(count);
+  }
+
+  public void incNumKeyTrashDeleteFails(int count) {
+    numKeyTrashDeleteFails.incr(count);
+  }
+
   public void incNumKeyDeletes() {
     numKeyOps.incr();
     numKeyDeletes.incr();
   }
 
+  public void incNumKeyDeletesWithoutOps() {
+    numKeyDeletes.incr();
```

Comment:
Could it be more clear if we introduce a numDirDeletes metrics, and rename this to incDirDeletes?

Reply:
The current deletion Metrics did not distinguish Key, File, and Directory, which might require additional changes. My modifications here are just to fix a metrics issue; otherwise, the test cases would fail.